recipes/cunit/build.sh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Can you add shebangs to all `build.sh` please? I guess that conda does not mind, but it looks nicer with them :smile: